### ISSUE TITLE
Bitcoin.org plans for 2021

### ISSUE THREAD

**Issue by Cobra-Bitcoin** [bitcoin-dot-org/Bitcoin.org#3559]
Hey guys, just thought I would share some of the potential upcoming improvements for bitcoin.org in 2021. These improvements will make bitcoin.org offer more value to new users, and hopefully breathe fresh life into the overall project (because ultimately there's only so many contributions people can make to just a simple static website, and many times, those contributions are just people adding links to their own services). So here's a list of the upcoming new stuff:

**Our own bitcoin wallet**:
I've been working on creating a bitcoin wallet for new users. This is intended to be the official bitcoin.org wallet and is designed from the ground up for beginners. Currently the [mobile version](https://github.com/bitcoin-dot-org/BitcoinWalletMobile) is mostly finished, and the [desktop version](https://github.com/bitcoin-dot-org/Bitcoin-Wallet) needs a bit more work, but it'll be fully done soon. 

Most of the visitors to our site are just looking for a basic starter wallet, but our wallet selection wizard asks a bunch of questions that no new user will fully comprehend, and then most of the time recommends something like Electrum, or the Schildbach Android wallet. These are great wallets, but they both look dated aesthetically, have too many features that our visitors don't need, and the Schildbach wallet in particular doesn't even support bip39 mnemonics. Some of the other wallets we recommend also support altcoins. I think our visitors are best served if they get a well designed wallet, with bip39 support, no unnecessary complex features, and can be downloaded on basically every platform. So that's what I've tried to create here.

The plan hopefully is to adjust the choose your wallet page, with new users being recommended our own basic wallet. The wallet will then monitor the user's balance, and if it goes above a certain value, it will prompt the user to get a hardware wallet and link to an "advanced" version of the choose your wallet page with only a selection of different hardware wallets. We basically let them get their feet wet on our basic wallet, where it'll be really hard to screw up, and then once they have accumulated enough BTC, we forward them to something more secure.

**Translation portal**:
One of this project's major strengths is our excellent translation team, definitely the best volunteer translation team in the entire wider cryptocurrency community. No other Bitcoin educational resource can even compare to bitcoin.org when it comes to internationalization. So I want to improve on this even more, and allow us to translate content even quicker. I want us to be able to write high quality educational Bitcoin articles on many different topics, and have them translated into multiple languages immediately days after.

To achieve this we need to build out our own translation portal with built in Bitcoin rewards, instead of relying on Transifex. Users will register on translate.bitcoin.org, enter some basic information like their email address, languages, and a BTC address, and then immediately start translating or receive notifications when there's new strings to translate. Once translations are reviewed, we would have a bot automatically create and merge the pull request. We can then use this platform to increase our ability to produce local language content quickly, to avoid long periods of time where new content is only in English and produces a bad experience of English mixed in with local language content.

**Funding a Bitcoin developer(s)**:
It seems lots of companies and entities are now funding open source Bitcoin developers (whether they work on Bitcoin Core, or on other Bitcoin based projects), but most of the time the money is coming from private companies or associated wealthy individuals. There doesn't seem to be a way for the Bitcoin community to collectively contribute to a developer fund and have the money go directly to developers. If it exists, it definitely isn't raising enough money nor is it well known. I think headlines like "the Bitcoin community raises $X for developers" makes us sound and look better than some private company throwing $100k at developers. Given all the people in the community who got wealthy off BTC, the least we can do is give back.

We can definitely fundraise for this, and ideally raise a lot of money for this goal. Considering we've had instances of people donating $100k+ in the past without us even really pushing for donations all that much, it seems likely we can raise enough to at least fund one developer for a year, which is the minimum goal here.

**Closing thoughts**:

I also hope to make lots of smaller improvements to the site myself, and continue to improve and grow the site so it continues to be the best educational resource for new users. This year you will see even more thought put into how people completely new to Bitcoin can walk away from bitcoin.org with useful knowledge and good tools to get them started in the community.

I also want to move away from people treating the site as a billboard for their projects. They come in, add a link to their site or service, and then never make another contribution ever again. I'm sick of it, and we're going to be a lot more strict on what types of stuff we promote on the site.

Excited for 2021 and seeing how Bitcoin continues to grow!

**Comment by TonyXhufi** [comment]
@Cobra-Bitcoin Great vision and looking forward contributing further in 2021. Re-Bitcoin Developers Fund happy to assist here...At the moment I am speaking to new comers in the space (institutions & HNW) + existing BTC holders so happy to assist with structuring the fund, legal, operations, marketing & business development. I agree if you put a proper structure/ process in place we would be able to raise awareness & funds to support development going forward in more efficient way. Let me know if you want to discuss further thanks Tony

**Comment by pavlenex** [comment]
@Cobra-Bitcoin Ambitious plans. I like the idea for wallets. I'm unsure if you're aware but if design/ux/ui help is needed (though you seem to have that one figured out), [bitcoin.design ](https://bitcoin.design)community can be of great help. Happy to connect the two communities.

Translation portal seems a bit be too ambitious. Transifex is working and has an established process and growing community. By introducing instant financial rewards for translations, there's a possibility of lower quality or even random translated content just for the reward. I'm just scared that by switching platforms, we'll loose a big number of existing contributors who are on Transifex not just because of bitcoin.org Being in translators community quite a bit, I think @Komodorpudel is doing great job on organizing things, has this been discussed with him?

One thing I'd personally like to see focus being on is the content. There's quite a bit of outdated and incorrect information on various pages.

I'm biased on this one, but if help is needed for setting a fundraising platform, happy to assist on setting up a BTCPay Server which already has this [built in](https://docs.btcpayserver.org/Apps/#crowdfunding-app).

I'm glad you're taking initiative finally, a bit sad that @wbnns is no longer here, he's absence is visible, but hopefully some new people can appear and help out, I am happy to help in certain areas (will look through the issues), but time-constrained.

**Comment by Komodorpudel** [comment]
@Cobra-Bitcoin my opinions on this:

**Regarding "Our own bitcoin wallet":**
I like the idea, though this should be implemented in a way that bitcoin.org stays neutral, e.g. the new wallet gets suggested if criteria x,y,z is met while all information on any other wallet is still as easily accessible as it currently is.

**Regarding "Translation portal":**
I have to agree with @pavlenex here.
While using the Transifex web interface is not perfect, it is definitely working. And it is also possible to setup an automated link between GitHub and Transifex to streamline the process. AFAIK there should also be other providers that allow to pay out rewards/bonuses (at least that's what I remember from 2017) if it is important to integrate a way to pay out bonuses, but switching the platform should hurt long-term for the reasons already mentioned by @pavlenex .

Also, the reason paying out bonuses was done retroactively and without prior announcement was already  mentioned by @pavlenex: Quality could suffer as people come just for the reward/bonus.

Now, beyond that, Machine Translating is getting better and better (and is actually already enabled for bitcoin.org via my AWS account) and it wouldn't surprise me that in the foreseeable future only reviewers are needed to check machine-generated translations.

Thus, while we definitely could improve the current process, I do not think creating a new translation portal dedicated to bitcoin.org is worth the effort.

**Regarding "Funding a Bitcoin developer(s):":**
100% agree, but the important question would be: Which developer? It would be necessary to setup a committee to decide which developer to fund, and in every case it is highly likely that bitcoin.org, a website dedicated to informing about Bitcoin, gets drawn into Bitcoin politics (which kind of happened before and wasn't too great).

Another question to think about: Why should people donate money to bitcoin.org only that that money is then used to fund a certain developer? Those individuals could just fund their favorite developer directly. So what value-add does donating to bitcoin.org provide over funding a developer directly?

**Website maintainer:**
@wbnns absence is indeed visible, and before starting other bigger projects I really feel like bitcoin.org needs a dedicated maintainer first.

**Other thoughts on content on the website:**
In addition to outdated content on the website: Bitcoin's narrative is shifting. Bitcoin.org displays Bitcoin as a digital currency, but as we all now, in its current state it does not really work like that. Which is not a real problem as people are working on solutions (be it Lightning or Wrapped BTC on another chain or whatever) and in the meantime it works perfectly fine as superior alternative for gold. But it might be worth it to either (a) change content on the website to address this or (b) provide a pop-up to inform on the current situation. We have a lot of new users coming in and it is important to present them with the facts :)

**Comment by pavlenex** [comment]
Funding a developer would require either a lot of research or simply one man to decide. Alternatively, funds can be given to an organization, but that's not a solution to avoid some of the issues @Komodorpudel brought up. 

Another possible idea, is to fund someone to work on bitcoin.org full time. I'm sure that there are contributors here that could focus their efforts on certain parts, for example developer documentation.

Here's a quick list of actionable things that came across my mind while thinking about how we can kick things off. I assume lots of contributors here get put off a bit since there's a lot of unreviewed PR's. Please take this as a suggestion. I'm just trying to brainstorm so that whoever is in charge can get some ideas (hopefully)

- Enable GitHub discussions  [example](https://github.com/btcpayserver/btcpayserver/discussions)
- Create Issue template to distinguish issue from questions/suggestions [example](https://github.com/btcpayserver/btcpayserver/issues/new/choose)
- Close stale/irrelevant issues and try to keep issues for actionable items
- Decide on Events 
- Try to have review sessions for pull requests
- Try to have community meetings [example](https://github.com/BitcoinDesign/Meta/issues/)
- Define processes on how people can contribute more clearly (label issues, document, etc)
- Review pages that require updating and open issues outlining step by step what user who wants to contribute needs to do (clearer processes and tasks)
- Think of communicating bitcoin.org vision through a roadmap

Personally I feel like awaking the community and organizing contributors a bit better would go a long way instead of funding, which can come at a later point and only to people who've been on a project for years.

Just suggestions that in general would work for any FOSS project. Not trying to make decisions here, or influence them in any way.

**Comment by Cobra-Bitcoin** [comment]
We could definitely keep Transifex while also having some way to quickly produce translated content. So we can try to get the best of both worlds. It doesn't necessarily need money, but maybe visibility, e.g. by pointing translators to new content and giving them a heads up that translation for that content would be extra appreciated. Anything to reduce long delays between new English content being produced and that content getting translated into more languages, because right now, it can take up to a year for any new content we add to get translated and actually read by new users in their local language.

With the regards to the wallet and neutrality, let's keep in mind we don't owe anybody free promotion here. If we can do better and offer newbies something designed for them, then let's do it. It also expands the project significantly, and maybe people that come to contribute to the wallet will also be tempted to contribute to the website, and vice versa. We would still promote other wallets, but only ones that would be more suitable for experienced users, like anonymity focused wallets, hardware wallets, and LN wallets. If the user is new and looking for a simple wallet, we'll just recommend ours in place of something like Schildbach wallet.

Agreed on dropping events, way too spammy, and there's only a handful of 'real' Bitcoin conferences each year anyway.

> In addition to outdated content on the website: Bitcoin's narrative is shifting. Bitcoin.org displays Bitcoin as a digital currency, but as we all know, in its current state it does not really work like that.

I think it's problematic for bitcoin.org to jump too deep into the narrative of BTC being digital gold and store of value. It inevitably requires that we suggest Bitcoin will always hold or increase its value, and that's not necessarily true, and we would basically be offering our visitors financial advice if we were to do that. The digital currency narrative is at least consistent with the history of the site, and if Lightning Network eventually builds momentum, it will still hold true.

It may be possible to change the site to conform to the new narrative, but I just don't know how we could do it without coming across as a get rich quick scheme and giving financial advice, especially if we pitch Bitcoin as an investment?

Maybe we can get away from the need for a sole "maintainer"? I think if we reduce all these self-promotion pull requests, remove events, and try to automate the Transifex translations somehow, it'll be a lot less work to keep on top of things. And Craig is doing a good job keeping up with the wallet reviews, so that's being taken care of very well. I've been quite busy working on the wallet, but once that's stable and released, I'll have more time to focus on the maintenance stuff.

I like the idea of a list of pages that need updating that we can focus on.

**Comment by pavlenex** [comment]
I am fan of "if it's not broken don't fix it", so I'd still suggest to re-consider shifting efforts/focus from building a translation platform into something else. I'm not sure if you used it, but for us who translate, it's a full-featured platform with lots of built in stuff that's helpful, so it will be hard to replicate the benefits it offers without having a team of people who work on it full time. IMO there are way more important things dev effort could be focused.

As far as shifting the narrative goes, I remember a few years ago when I started reading bitcoin.org I always appreciated it didn't force me into buying bitcoin and it always warned about the experimental nature and the importance of not investing what you can't afford to lose.

That said, BItcoin will always want want a person want it to be. I agree shifting can be problematic, but why can't we use both? Bitcoin is currency and a store of value. We can perhaps add a bit of content about digital gold and store of value alongside text we already have.

Maintainer is needed to merge PR's that are reviewed and approved. For example the PR about dark mode #3204 took a lot of work, but in the end nobody even reviewed it and it's staled. So if bitcoin.org needs more contributors, their efforts need to be valued and someone needs to take time to review and eventually merge if accepted.

So, I'd suggest to focus efforts on cleaning things up, waking up contributors community and making their time valued, improving the review process and determining one-two people who've been active [contributors](https://github.com/bitcoin-dot-org/Bitcoin.org/graphs/contributors) with a long-trail of contributions to the site as maintainers and drafting up a plan on how to improve content on existing pages as the next few steps.

**Comment by Cobra-Bitcoin** [comment]
@pavlenex OK, maybe we can set aside the translation portal stuff then, as you say Transifex seems to be working for you guys and I don't want to ruin the existing workflow.

I don't mind taking on more maintenance activities myself, naturally some PRs get missed (I just responded to the one you mentioned, thanks for pointing it out, the "changes requested" label threw me off), but I think strongly discouraging all of the spammy "add link to our service" type PRs would go a long way in cleaning up the project and making general day-to-day activity easier. To that end we can do the following:

- We have one issue titled "Recommend an exchange to add": and users can reply to it and suggest to us exchanges to add, and every 3 months we review, and add exchanges that look OK
- Remove events
- Once we list our wallet, we use that as the "starter wallet" for new users, and the only new wallet submissions we process beyond that point are specialized wallets designed for more advanced users, like anonymity focused wallets, or hardware wallets

Let's also keep in mind that at the end of the day, bitcoin.org is just a static website, and will always attract somewhat less interest and novel contributions than actual software projects. And it's not a project that requires a lot of constant change, there can be one month with a very dramatic change, followed by half a year with no change. To that end we should be comfortable with some level of stasis. 

> That said, BItcoin will always be what a person wants it to be. I agree shifting can be problematic, but why can't we use both? Bitcoin is currency and a store of value. We can perhaps add a bit of content about digital gold and store of value alongside text we already have.

I'll give you an example of how this could be hard, if you read this page: https://bitcoin.org/en/you-need-to-know#volatile we say the following: 

> The price of a bitcoin can unpredictably increase or decrease over a short period of time due to its young economy, novel nature, and sometimes illiquid markets. Consequently, keeping your savings with Bitcoin is not recommended at this point. Bitcoin should be seen like a high risk asset, and you should never store money that you cannot afford to lose with Bitcoin. If you receive payments with Bitcoin, many service providers can convert them to your local currency.

If you ask me, basically everything except the "illiquid markets" sentence is true in that paragraph. And this is good advice (for new users! obviously we all hodl and keep our life savings in BTC). It would be really strange if some other part of the site were to say Bitcoin is something like a "store of value", when that paragraph makes clear you can lose a lot of money quickly. 

Maybe we can pivot to something like: "Bitcoin is a new type of money and financial network, Bitcoin means many things to different people. For some, Bitcoin is a new way to make payments digitally, and for others, Bitcoin is a savings tool. Whatever you use Bitcoin for, it's a groundbreaking new technology.". 

Basically we mention Bitcoin can be money, is a network, and that people use it for different things; without necessarily encouraging or pushing one thing in particular. The digital gold stuff can be framed as if we're talking about other people, e.g. sentences like "some people even believe in Bitcoin so much, they're willing to put all their money into it, but this is something you should decide for yourself". That could work, but you'd need to be really careful to not cross into giving financial advice.

**Comment by pavlenex** [comment]
> I don't mind taking on more maintenance activities myself

This is good news. Project needs an active maintainer. 

> I think strongly discouraging all of the spammy "add link to our service" type PRs would go a long way in cleaning up the project

I 100% agree. Clean-up is much-needed.

A collective issue as you suggested, or alternatively create an [issue template ](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository) and then redirect events submission towards a discussion in the `events` category where these topics can be discussed and voted on (discussions plan to have voting feature in the future, [example](https://github.com/github/feedback/discussions)) We do that for feature requests on btcpay, [example](https://github.com/btcpayserver/btcpayserver/issues/new/choose))

The friction on issues  template prevents random questions and adds templates for issues themselves which provides a proper  format for questions, bug reports, etc.  That kind of structure makes it easier for contributors to answer/tackle issues if they are properly explained. This in addition to proper use of labels combined with GitHub Projects (roadmap example, short term plan example) really adds transparency to the whole process and makes it clearer to contributors on what are priorities to focus on. 

> I'll give you an example of how this could be hard, if you read this page: https://bitcoin.org/en/you-need-to-know#volatile we say the following.

Yes this is quite solid example on why combining may not work as easy as I originally thought off.

**Comment by David200308** [comment]
I already finished the Chinese Version translation and fork on https://github.com/David200308/BitcoinWalletMobile/blob/patch-1/lang/zh.ts, wish some people can help me to check it, and fix it, wish make more better.

**Comment by hassanhamraei** [comment]
dear Cobra Please contact ME , I need your help.

**Comment by Komodorpudel** [comment]
@Cobra-Bitcoin @pavlenex 

Alrighty guys, I got another, much more basic suggestion before we start with anything bigger:

https://trello.com/b/cpQ5fgBS/bitcoinorg

A public Trello board where relevant tasks are organized.

I always felt that bitcoin.org was missing an overview of "what the heck is going on currently", which made it difficult to coordinate (This was kind of fine when Will was around because he was the nucleus, but now it is a bit more "decentralized"). Also, this way, it would be easy for the community to see what is going on. I added some example tasks to see how it would look like.

@Cobra-Bitcoin if you are interested in working with Trello, I could invite everyone regularly working on bitcoin.org (this way, we could also avoid our monthly chats on Twitter :) )

**Comment by pavlenex** [comment]
@Komodorpudel This is cool, but there's a native issue within GItHub called projects that makes this way more natively integrated and imo easier. I've noticed it's been utilized in the past. Don't have a strong preference for this if it's easier for you guys to use Trello that's fine, just wanted to mention that [integrated solution exist](https://docs.github.com/en/github/managing-your-work-on-github/about-project-boards).

**Comment by Komodorpudel** [comment]
> @Komodorpudel This is cool, but there's a native issue within GItHub called projects that makes this way more natively integrated and imo easier. I've noticed it's been utilized in the past. Don't have a strong preference for this if it's easier for you guys to use Trello that's fine, just wanted to mention that [integrated solution exist](https://docs.github.com/en/github/managing-your-work-on-github/about-project-boards).

Ohhh dear lord, completely missed this one. I only remembered how it was used for the redesign and was like "Nahhh", but this 100% makes sense.

**Comment by hassanhamraei** [comment]
Dear Sir

my name is hassan and I'm from iran , i got married if and i have big
problem with my Costs and expenses and i know you are a
billionaire because of you Effort and creativity so i ask you to send me a
few bitcoin for start my marriage living

**Comment by Komodorpudel** [comment]
@Cobra-Bitcoin push

**Comment by Cobra-Bitcoin** [comment]
I agree with keeping everything in Github whenever possible, we've already used the project board feature for the re-design before, and that seemed to work out well.

I'll make a board on Github using the stuff on @Komodorpudel's Trello board as a starter, and we can expand from there.

**Comment by Cobra-Bitcoin** [comment]
OK, the project board "to do" is up: https://github.com/bitcoin-dot-org/Bitcoin.org/projects/2. We can use that to keep track of things.

Also, as most of you have heard, we're facing some legal challenges from a certain notorious person for hosting the Bitcoin whitepaper, we never seem to be able to escape drama it seems. Hopefully that won't prevent things from continuing to improve with this project.

**Comment by Komodorpudel** [comment]
@Cobra-Bitcoin Perfect, that's a good start!

**Comment by cryptohakka** [comment]
I have some thought about "Reworking translation team page".
I tried to define contribution score as following calculation and checked how many languages and translators can be listed if I set two types of borders and max 3 translators for each language.

Contribution score = new_total + (edit_total+review_total)/2

I prioritized new_total because translating new sentences is more difficult than editing or reviewing them.
According to two types of boarder, numbers of languages and translators are like below.

Contribution score >= 1,000; languages=49, translators=120
Contribution score >= 10,000; languages=31, translators=53

In my thought, the latter is better because some languages' main page of the former has not been completely translated.
It takes about 1 hour if I PR based on the latter. I appreciate if you guys have any thought about it.

**Comment by Komodorpudel** [comment]
> I have some thought about "Reworking translation team page".
> I tried to define contribution score as following calculation and checked how many languages and translators can be listed if I set two types of borders and max 3 translators for each language.
> 
> Contribution score = new_total + (edit_total+review_total)/2
> 
> I prioritized new_total because translating new sentences is more difficult than editing or reviewing them.
> According to two types of boarder, numbers of languages and translators are like below.
> 
> Contribution score >= 1,000; languages=49, translators=120
> Contribution score >= 10,000; languages=31, translators=53
> 
> In my thought, the latter is better because some languages' main page of the former has not been completely translated.
> It takes about 1 hour if I PR based on the latter. I appreciate if you guys have any thought about it.

FYI: I am currently discussing this with @cryptohakka and a PR should happen soon :)

**Comment by mateusnds** [comment]
Hello, Cobra. Nice job.

I am one of the translators, I noticed that the pages in Portuguese still point to links in English, which can confuse users, do you think we can "fix" this?

**Comment by lodembeep** [comment]
@mateusnds You should be able to do the changes in transifex, use the filters to get the links, then change en to your language code.

@Cobra-Bitcoin shouldn't  #3545 be added to the board as a suggested project ?
Also i suggest adding another maintainer, even temporarily in order to clean the PRs and issues 😊

**Comment by hassanhamraei** [comment]
MY ETH ADDRESS: 0x80141c3E2E772184Efb9618480Bb950544E457DB

MY BNB ADDRESS : 0x80141c3E2E772184Efb9618480Bb950544E457DB
 MY BITCOIN ADDRESS : bc1q75dptu49aj6axh8zw9pl3yqe5crzajn7g6jlg7
